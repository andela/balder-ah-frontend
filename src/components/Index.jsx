import React from 'react';

const Index = () => (
  <div className="App">
    <header className="App-header">
      <p>
        {'Welcome to Author\'s Haven by Team Balder!'}
      </p>
    </header>
  </div>
);

export default Index;
