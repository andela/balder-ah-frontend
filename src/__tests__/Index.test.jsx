import React from 'react';
import { cleanup } from 'react-testing-library';
import 'jest-dom/extend-expect';

import renderWithRouter from '../__mocks__/helpers';
import Index from '../components/Index';

afterEach(cleanup);

describe('<Index />', () => {
  test('should render correctly', () => {
    const { container } = renderWithRouter(<Index />);
    expect(container).toMatchSnapshot();
  });
});
