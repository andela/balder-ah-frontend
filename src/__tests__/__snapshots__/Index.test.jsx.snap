// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> should render correctly 1`] = `
<div>
  <div
    class="App"
  >
    <nav
      class="nav"
    >
      <div
        class="navbar"
      >
        <div
          class="brand"
        >
          <a
            href="/"
          >
            Authors Haven
          </a>
        </div>
        <div
          class="nav-links"
        >
          <ul
            class=""
            id="nav-mobile"
          >
            <li>
              <a
                href="/create-article"
              >
                Create article
              </a>
            </li>
            <li>
              <i
                class="small icon material-icons"
              >
                search
              </i>
            </li>
            <li>
              <i
                class="small icon material-icons"
              >
                menu
              </i>
            </li>
          </ul>
        </div>
      </div>
    </nav>
    <header
      class="App-header"
    >
      <p>
        Welcome to Author's Haven by Team Balder!
      </p>
    </header>
  </div>
</div>
`;
